pkg/util/csi/volume_snapshot_test.go

```diff
@@ -1684,28 +1684,28 @@ func TestWaitUntilVSCHandleIsReady(t *testing.T) {
 			},
 		},
 		{
-			name:        "waitDisabled should not find volumesnapshotcontent volumesnapshot status is nil",
+			name:        "waitDisabled should not find volumesnapshotcontent when volumesnapshot status is nil",
```

Comment:
This is separate from dupword right? It's a good change just wanna confirm.

Reply:
Oh I see.. probably from `volumesnapshotcontent volumesnapshotcontent` several lines below.